components/dashboard/src/components/SolidCard.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+function SolidCard(p: { className?: string; children?: React.ReactNode }) {
+    return (
+        <div
+            className={
+                "flex rounded-xl text-base w-72 h-64 px-4 bg-gray-100 dark:bg-gray-900 text-gray-600 dark:text-gray-600" +
```

Comment:
**suggestion**: Skipped to mentioned this change in https://github.com/gitpod-io/gitpod/pull/9343#discussion_r853372863. 😅 

```suggestion
                "flex rounded-xl text-base w-72 h-64 px-4 bg-gray-100 dark:bg-gray-800 text-gray-600 dark:text-gray-600" +
```